.travis.yml

```diff
@@ -9,4 +8,12 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
+
+matrix:
+  include:
+    - rvm: 2.6
+      env: RUBYOPT="--jit"
+  allow_failures:
+    - env: RUBYOPT="--jit"
```

Comment:
remove this since it passes ?

Reply:
The `--jit` one fails, quite badly...  I didn't want to block green for the whole project since JIT is still experimental.

https://travis-ci.org/grosser/parallel/jobs/518878930

Reply:
oh yeah ... idk what I looked at ...